extras/kms/errors.go

```diff
@@ -20,6 +20,9 @@ var (
 	// ErrKeyNotFound represents that no key was found
 	ErrKeyNotFound = errors.New("key not found")
 
+	// ErrKeyVersionNotFound represents that no key version was found
+	ErrKeyVersionNotFound = errors.New("key version not found")
```

Comment:
Not convinced about the need for this, since it is technically another breaking change on the API. Happy to remove it if we think it'll cause more problems than it'll help. Added it because we may want to be able to tell the difference between key not found and key version not found on the API.

Reply:
Will remove it